website/content/en/docs/migration/version-upgrade-guide.md

```diff
@@ -985,6 +1090,69 @@ func serveCRMetrics(cfg *rest.Config, operatorNs string) error {
 
 **NOTE**: For more information check the PRs which are responsible for the above changes [#2606](https://github.com/operator-framework/operator-sdk/pull/2606),[#2603](https://github.com/operator-framework/operator-sdk/pull/2603) and [#2601](https://github.com/operator-framework/operator-sdk/pull/2601).
 
+**Support for watching multiple namespaces**
```

Comment:
I do not think that we need to add it as well. It is optional. However, if it is added we need to make clear. 
 ```suggestion
**(OPTIONAL) Support for watching multiple namespaces**
```

Reply:
Yes, I had it in my mind to mark both changes as optional but totally forgot about it. Thanks.

Reply:
Same as above ^

Reply:
I'm proposing this section for 0.16.x as optional. As far as I can see from changelog, this was introduced in 0.16. Is there something I'm missing here?

Reply:
The users can upgrade their project without applying these changes introduced in the scaffolded. 
Alll will still be working without that, in this way it is an optional change/improvement in the process to migrate the project use an upper version.

Reply:
Yep, marked it optional.